### ISSUE TITLE
Support pnpm

### ISSUE THREAD

**Issue by JohnCampionJr** [DevExpress/devextreme-cli#288]
# Feature request

 **Package versions you currently use:**

     devextreme-cli version: 1.2.10

 **Description:**
Does not work if using pnpm package manager.  If I delete node_modules and use npm it works fine.

Throws error:
```
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at getDevExtremeInfo (<redacted>/node_modules/.pnpm/devextreme-cli@1.2.10/node_modules/devextreme-cli/src/themebuider.js:130:29)
    at getDevExtremeVersion (<redacted>/node_modules/.pnpm/devextreme-cli@1.2.10/node_modules/devextreme-cli/src/themebuider.js:139:71)
    at Object.runThemeBuilder [as run] (<redacted>/node_modules/.pnpm/devextreme-cli@1.2.10/node_modules/devextreme-cli/src/themebuider.js:181:40)
```

 **Preferred Solution:**
Should work fine with pnpm as it does with nom / yarn

 **Alternatives:**
None besides being required to use nom or yarn

**Additional Context:**

**Comment by babich-a** [comment]
Hi
We have a similar issue with npm/yarn. We'll check a pnpm package manager after the fix.

**Comment by babich-a** [comment]
Hi @JohnCampionJr . We have released v1.2.11 of the devextreme-cli package with the bug fix. I checked pnpm with your scenario (delete node_modules and run `pnpm install`), and it works for me. If you have more complex scenarios in which it doesn't work, feel free to contact us.

**Comment by JohnCampionJr** [comment]
Confirmed to work property; thank you!
